_posts/2020-03-15-learning-to-share-accomplishments-2.md

```diff
@@ -0,0 +1,214 @@
+are usually NDAs or not publicly available). So I hope this post helps with that as well.
+
+### Best Team: Data Pipeline Team
+
+This team was created to replace a $1M/year third-party product with an in-house solution. The third-party product was 
+expensive, a pain to set up, didn't scale well, and the company only used about 10% of it. Ours was built for this 
+specific solution, scaled with the click of a few buttons, and cost closer to $1000/month (a VAST savings). 
+
+**What did I Do?**
+
+* I was the generalist on this team. I felt like I was the glue that when given a random problem to solve, thing to 
+  research, or item to build, I just did it. This work often helped fix some problems that would have delayed our 
+  release or fix an issue that would have popped up later on. 
+* I figured out the best way to do an AWS-based service queue system for our monitoring microservice. I did the 
+  research by looking at all of AWS's types of queueing systems and their costs of running. I later implemented a basic 
+  prototype, and then wrote the final code to convert our monitoring service to push to it.
+* We had a broken build for our client executable that wasn't building separate versions correctly. I wrote some shell 
+  scripts that fixed Travis CI (for our Mac version) and AppVeyor (for our Windows version) to get both working. It 
+  then built and uploaded to AWS S3 buckets in the correct versioned structure.
+* I contributed at least one feature to the code base of every Scala microservice we had when I had no prior Scala 
+  experience. (It's similar to Haskell, which I had done a project on in the past, so I picked it up quickly.)
+* A teammate couldn't get one of our services working in Scala so they rewrote it all in Ruby over a weekend. We ended 
+  up keeping that, but it was built to run locally. After that person quit shortly after, I Dockerized it, wrote the CI 
+  scripts to deploy it to AWS, and got it to work with AWS's Aurora databases (as opposed to a local Postgres install). 
+  I later ended up resolving at least 85% of the security flaws that came up in a security audit on it. I did this 
+  without prior Ruby or Ruby on Rails knowledge, which I picked up quickly).
+* When I was hired, they wanted me to start work immediately and relocate to Pittsburgh as I could. I started work 
+  within about two weeks of the offer, and spent the first two months working remote out of Kansas City (where the time 
+  zone was an hour earlier than my team). I also was packing to move at the same time I did this work.
+* From day one, I was given ownership of two of the 8-10 microservices our project had.
+* I wrote and modified existing Terraform scripts to add additional deploy and containerization features for some of 
+  our microservices. (I had never used Terraform before either.)
+* After our original manager was fired and a new one was brought in, I regularly checked in on and mentored the junior 
```

Comment:
I like the way you bring in this "took up manager tasks" part too. A lot of people find they can't program and manage but there you were, and it wasn't even your job. Just you being a good egg. You deserve to get to talk about that.